packages/types/src/interfaces/staking/definitions.ts

```diff
@@ -1,6 +1,18 @@
 // CENNZnet specific staking types, the rest will be resolved from @polkadot/types
 
 export default {
+  rpc: {
+    accruedPayout: {
+      description: 'Retrieves the currently accrued reward for the specified payee (validator or nominator)',
+      params: [
+        {
+          payee: 'payee',
+          type: 'AccountId',
```

Comment:
we might need to fix this on node side also
```suggestion
          stash: 'stash',
          type: 'AccountId',
```